SCF/SCF-Session8-ZooProblem/Zone.java

```diff
@@ -0,0 +1,79 @@
+import java.util.*;
+
+public class Zone {
+
+	private String animalCategory;
+	private int maxCage;
+	private List<Cage> cages = new LinkedList<Cage>();
+	private boolean hasPark;
+	private boolean hasCanteen;
+	private int numberOfCagesLeft ;
```

Comment:
No need for this, as it doesn't make sense that you have cages but no animals inside.